src/components/Clips.js

```diff
@@ -179,16 +186,53 @@ class Clips extends React.Component {
     this.setState({modal: null});
   }
 
+  ClipsList() {
+    let clips;
+    if(this.props.playlistIndex !== undefined) {
+      clips = this.props.formStore.currentLocalizedData.playlists[this.props.playlistIndex].clips;
+    } else {
+      clips = this.props.formStore.currentLocalizedData.assets[this.props.storeKey];
+    }
+
+    if(this.state.activeFilter) {
+      clips = clips.filter(clip => clip.displayTitle.includes(this.state.activeFilter));
```

Comment:
A couple things about the filter:
- filters should work on at least both title and slug, and probably also type. ideally anything displayed should be filterable.
- make sure to lowercase both filter and target whenever you filter (and when calling lowercase make sure the target is defined, e.g. `(clip.displayTitle || "").toLowerCase()` to avoid avoid errors when data is bad.